Podcast/TagCard.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TagCard.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+//suggestion card based off tags user has liked
+class TagCard: EpisodeCard {
+    
+    var tagName: String
```

Comment:
this should use a Tag class instead of a string, and this variable should just be called "tag"